### ISSUE TITLE
Using static ignores Obsolete attribute

### ISSUE THREAD

**Issue by marek-safar** [dotnet/roslyn#10639]
**Version Used**: 1.2.0.60317

**Steps to Reproduce**:
1. Compile following program

```
using System;
using static D;

[Obsolete ("Broken class", true)]
static class D
{
    public static void Foo ()
    {

    }
}

class Test
{
    public static void Main ()
    {
        Foo ();
    }
}
```

**Expected Behavior**:

Compilation error

**Actual Behavior**:

No error

**Comment by jaredpar** [comment]
Thanks for reporting this issue!

**Comment by gafter** [comment]
Please target this to milestone 2.1.

**Comment by OmarTawfik** [comment]
Fixed by #16472
